### ISSUE TITLE
Book Swap

### ISSUE THREAD

**Issue by timoxley** [campjs/campjs-iii#20]
Same as last Camp:

Bring your old Tech books! Stop lining your shelves with books you won't read again because you think it makes you look smart or fancy. They're a pain in the ass to move around, and they're probably a fire hazard. Give them to someone who will read them!

Bring as many books as you can, no php books though.

**Comment by timoxley** [comment]
Also note: if you collected a book at the previous camp, please return it to the collection at the upcoming camp, _especially if you haven't read it yet_. I gave away my entire library at the last event, hoping to see at least some of them return and get passed-on.

I'm wondering if we should've implemented some tracking at the last event, so even if people aren't attending an upcoming campjs, they can still be hassled to mail their books back to be recirculated.

**Comment by sporto** [comment]
Worth announcing this on tweeter and emails

**Comment by timoxley** [comment]
Agree. Done https://twitter.com/campjsnews/status/461379282728525824

**Comment by stennie** [comment]
Maybe worth creating some CampJS labels & tagging for [BookCrossing](http://www.bookcrossing.com) or a similar service (if there is one)?

**Comment by timoxley** [comment]
Not a bad thought, we'll look into it.

**Comment by mursya** [comment]
So cool! The idea is simply brilliant!
